crates/solvers/src/infra/dex/balancer/mod.rs

```diff
@@ -0,0 +1,151 @@
+//! Bindings to the Balancer Smart Order Router (SOR) API.
+
+use {
+    crate::{
+        boundary,
+        domain::{auction, dex, eth, order},
+    },
+    ethereum_types::U256,
+    std::sync::atomic::{self, AtomicU64},
+};
+
+mod dto;
+
+pub struct Sor {
```

Comment:
Already noted in a follow up PR (reviewed the stack backwards) but I think we should either spell out the acronym here or put a doc comment on this `struct` instead of the module because you would actually see that with IDE inline hints when using the type. (I believe the doc comments would not be displayed)

Reply:
Agreed. 0bcffecd411b53910a72df8d7b67955c567747ff